### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by rmrlangford**

### CODE REVIEW

File: input/fsh/profiles.fsh [hunk 1]
```diff
@@ -245,11 +244,39 @@ Parent: Observation
...
-* effectiveDateTime 1..1 MS  //Covid19DeathDate
\ No newline at end of file
+* status 1..1
+* code = $SCT#419099009
+* subject only Reference(Patient)
+* subject 1..1
+* encounter 1..1
+* effectiveDateTime 1..1
+* note.authorReference only Reference(Organization)
+* note 1..1
```
Comment (mariusnaude) [inline input/fsh/profiles.fsh:268]: Why is the note element required?

**Review by mariusnaude — COMMENTED** [review]
Thanks, I see verificationStatus now use the required binding codes and the outcomes has been refactored to individual obs profiles.

**Comment by rmrlangford** [comment]
as per MDS

File: input/fsh/profiles.fsh (hunk 1 above)
Comment (rmrlangford) [reply input/fsh/profiles.fsh:268]: as per MDS

**Pull request merged**